openstack/nova/templates/api-deployment.yaml

```diff
@@ -58,8 +58,13 @@ spec:
               value: "localhost"
             - name: STATSD_PORT
               value: "9125"
+            {{- if .Values.sentry.enabled }}
             - name: SENTRY_DSN
-              value: {{.Values.sentry_dsn | quote}}
+              valueFrom:
+                secretKeyRef:
+                  name: sentry
+                  key: {{ .Chart.Name }}.DSN.python
```

Comment:
I was expecting to find something in the secrets that would define this value. I cannot. Where does it come from?

Reply:
nevermind, found it now

Reply:
Although it's still in the old format, `sentry_dsn`, so i guess there's a secret change necessary as well?

Reply:
I don't get your questions. Do you mean `{{ .Chart.Name }}`? The `sentry` secret? The second is described in the commit-message.

Reply:
Ah via k8s secret. What is the rationale behind doing it that way? Why not a regular secret? I'm asking because it adds another source of information, and increases spread.

Reply:
We have a `sentry-operator` which creates the projects in sentry for us and provides the DSN to us via secret. The point is, that one can reinstall sentry without having to do manual processing for every service (creating the project in sentry, updating the secrets).

Reply:
i see.